cloud/jenkins/pgo_operator_eks_version.groovy

```diff
@@ -262,48 +262,25 @@ void runTest(Integer TEST_ID) {
             timeout(time: 90, unit: 'MINUTES') {
                 withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: 'eks-cicd', secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {
                     sh """
-                        cd ./source
-                        if [ -n "${PG_VERSION}" ]; then
-                            export PG_VER=${PG_VERSION}
-                        fi
-                        if [ -n "${PGO_OPERATOR_IMAGE}" ]; then
-                            export IMAGE=${PGO_OPERATOR_IMAGE}
-                        else
-                            export IMAGE=perconalab/percona-postgresql-operator:${env.GIT_BRANCH}
-                        fi
-    
-                        if [ -n "${PGO_PGBOUNCER_IMAGE}" ]; then
-                            export IMAGE_PGBOUNCER=${PGO_PGBOUNCER_IMAGE}
-                        fi
-    
-                        if [ -n "${PGO_POSTGRES_IMAGE}" ]; then
-                            export IMAGE_POSTGRESQL=${PGO_POSTGRES_IMAGE}
-                            export PG_VER=\$(echo \${IMAGE_POSTGRESQL} | grep -Eo 'ppg[0-9]+'| sed 's/ppg//g')
-                        fi
-    
-                        if [ -n "${PGO_BACKREST_IMAGE}" ]; then
-                            export IMAGE_BACKREST=${PGO_BACKREST_IMAGE}
-                        fi
-    
-                        if [ -n "${PGO_PGBADGER_IMAGE}" ]; then
-                            export IMAGE_PGBADGER=${PGO_PGBADGER_IMAGE}
-                        fi
-    
-                        if [ -n "${PMM_SERVER_IMAGE_BASE}" ]; then
-                            export IMAGE_PMM_SERVER_REPO=${PMM_SERVER_IMAGE_BASE}
-                        fi
-    
-                        if [ -n "${PMM_SERVER_IMAGE_TAG}" ]; then
-                            export IMAGE_PMM_SERVER_TAG=${PMM_SERVER_IMAGE_TAG}
-                        fi
-    
-                        if [ -n "${PMM_CLIENT_IMAGE}" ]; then
-                            export IMAGE_PMM=${PMM_CLIENT_IMAGE}
+                        cd source
+
+                        [[ "$OPERATOR_IMAGE" ]] && export IMAGE=$OPERATOR_IMAGE || export IMAGE=perconalab/percona-postgresql-operator:$GIT_BRANCH
+                        export PG_VER=$PG_VERSION
+                        export IMAGE_PGBOUNCER=$PGO_PGBOUNCER_IMAGE
+
+                        if [[ "$PGO_POSTGRES_IMAGE" ]]; then
+                            export IMAGE_POSTGRESQL=$PGO_POSTGRES_IMAGE
+                            export PG_VER=\$(echo \$IMAGE_POSTGRESQL | grep -Eo 'ppg[0-9]+'| sed 's/ppg//g')
                         fi
-    
+
+                        export IMAGE_BACKREST=$PGO_BACKREST_IMAGE
+                        export IMAGE_PGBADGER=$PGO_PGBADGER_IMAGE
```

Comment:
do we use it?

Reply:
I'm not sure. All I can see is [this](https://github.com/percona/percona-postgresql-operator/blob/587667ce8751b18df95f9329a597a485f57890dd/e2e-tests/vars.sh#L18), but then again, it is never used inside the repository.

Reply:
Done, I removed IMAGE_PGBADGER